City-Bureau/city-scrapers-wichita
# 🏗️ Build: WAMPO tabs mixin + 4 spiders

## What's this PR do?
Adds a mixin and three spiders to scrape meetings from the website of Wichita Area Metropolitan Planning Organization. Targets the following committees:

- Executive Committee
- Active Transportation Committee
- ICT Safe
- United Community Transit Coalition

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
\`\`\`
pipenv sync --dev
\`\`\`
2. Activate the virtual env and enter the pipenv shell:
\`\`\`
pipenv shell
\`\`\`
3. Run the spiders:
\`\`\`
scrapy list | grep 'wicks_wampo' | xargs -I {} scrapy crawl {} -O {}.csv
\`\`\`
4. Monitor the output and ensure no errors are raised.

5. Inspect the CSVs to ensure the data looks valid. You can view the "source" column of the rows to ensure the data matches the page it was extracted from.


## Are there any smells or added technical debt to note?
- We have to hardcode a lot of details because of spare meeting data. Good links to agendas & minutes though.
- Includes a mixin test and tests for 2 of the 3 spiders. That seemed like adequate coverage to me.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206641242761852
